INSTALL.md

```diff
@@ -0,0 +1,136 @@
+# Installing client-go
+
+## For the casual user
```

Comment:
I'm not sure that this approach will ever work.  The minute you start using any API objects you pull in apimachinery from your gopath.  I think the only *real* way to make this work is to flatten dependencies.

```
$ mkdir -p ~/tmp/go
$ export GOPATH=~/tmp/go
$ go get k8s.io/client-go/...
$ cd ~/tmp
$ cp go/src/k8s.io/client-go/examples/out-of-cluster/main.go
$ go build main.go
main.go:24:2: cannot find package "k8s.io/apimachinery/pkg/apis/meta/v1" in any of:
	/usr/local/Cellar/go/1.7.4/libexec/src/k8s.io/apimachinery/pkg/apis/meta/v1 (from $GOROOT)
	/Users/jbeda/tmp/go/src/k8s.io/apimachinery/pkg/apis/meta/v1 (from $GOPATH)
$ go get k8s.io/apimachinery/...
$ go build main.go
# command-line-arguments
./main.go:46: cannot use "k8s.io/apimachinery/pkg/apis/meta/v1".ListOptions literal (type "k8s.io/apimachinery/pkg/apis/meta/v1".ListOptions) as type "k8s.io/client-go/vendor/k8s.io/apimachinery/pkg/apis/meta/v1".ListOptions in argument to clientset.CoreV1().Pods("").List
```